app/build.gradle

```diff
@@ -85,17 +85,12 @@ android {
 
     buildTypes {
         buildTypes.each {
-            def serverBase = surveyProps['serverBase'].toString()
             def awsBucket = surveyProps['awsBucket'].toString()
+            def serverBase = "https://${awsBucket}.appspot.com"
```

Comment:
I would recommend to get this value from the `serverBase` as you had two lines above.  In the case of the older instances the `awsBucket` doesn't correspond to the `appId`